src/tests/functional/shared_test_classes/src/single_op/clamp.cpp

```diff
@@ -0,0 +1,51 @@
+// Copyright (C) 2018-2023 Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0
+//
```

Comment:
Please use the one style in copyright headers
```suggestion
// Copyright (C) 2018-2023 Intel Corporation
// SPDX-License-Identifier: Apache-2.0
//
```

Reply:
done